packages/client.api/src/index.ts

```diff
@@ -76,3 +76,40 @@ export type { LinkedType, LinkNames } from "./util/LinkUtils.js";
 export type { NOOP } from "./util/NOOP.js";
 
 export {};
+export type {
```

Comment:
Added a lot here, as mentioned in previous PRs, definitely should look through this and make sure we prune things we don't want to expose before releasing

Reply:
Nit can you group these with the the above exports, leaving a space with the `export {};` last and run cmd+shift+p organize imports?